src/equations/laplace_diffusion_3d.jl

```diff
@@ -0,0 +1,72 @@
+@doc raw"""
+    LaplaceDiffusion3D(diffusivity, equations)
+
+`LaplaceDiffusion3D` represents a scalar diffusion term ``\nabla \cdot (\kappa\nabla u))``
+with diffusivity ``\kappa`` applied to each solution component defined by `equations`.
+"""
+struct LaplaceDiffusion3D{E, N, T} <: AbstractLaplaceDiffusion{3, N}
+    diffusivity::T
+    equations_hyperbolic::E
+end
+
+function LaplaceDiffusion3D(diffusivity, equations_hyperbolic)
+    LaplaceDiffusion3D{typeof(equations_hyperbolic), nvariables(equations_hyperbolic),
+                       typeof(diffusivity)}(diffusivity, equations_hyperbolic)
+end
+
+function varnames(variable_mapping, equations_parabolic::LaplaceDiffusion3D)
+    varnames(variable_mapping, equations_parabolic.equations_hyperbolic)
+end
+
+# no orientation specified since the flux is vector-valued
```

Comment:
The Laplace 3D is just copied from 2D - Maybe @jlchan can explain?

Reply:
I think it may just be outdated - a previous version returned the flux vector, while this current version only returns an individual flux component. 

Would you mind deleting those comments?